modules/eco-poison-pill-operator-installation-cli.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+[id="installing-poison-pill-operator-using-cli_{context}"]
+= Installing the Poison Pill Operator by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the Poison Pill Operator.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Namespace` custom resource (CR) for the Poison Pill Operator:
+.. Create the following `Namespace` CR and save the YAML file, for example, `poison-pill-namespace.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: poison-pill
+----
+.. To create the `Namespace` CR, run the following command:
++
+[source,terminal]
+----
+$ oc create -f poison-pill-namespace.yaml
+----
+
+. Create an `OperatorGroup` CR:
+.. Create the following `OperatorGroup` CR and save the YAML file, for example, `poison-pill-operator-group.yaml`:
```

Comment:
You're not actually creating the CR until step 1b, so I'd rephrase this slightly:

```suggestion
.. Define the `OperatorGroup` CR and save the YAML file, for example, `poison-pill-operator-group.yaml`:
```